### PR TITLE
Edited

### PR DESCRIPTION
**Pull request by Paryax**
- [DEMO LINK](https://Paryax.github.io/layout_stars/)
- [TEST REPORT LINK](https://Paryax.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star-active.svg);
```
Comment (ivan-kadykalo) [inline src/style.css:12]: Use inactive star here

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  margin-right: 0;
+}
+
+.stars--0 .stars__star:nth-child(n),
+.stars--1 .stars__star:nth-child(n + 2),
+.stars--2 .stars__star:nth-child(n + 3),
+.stars--3 .stars__star:nth-child(n + 4),
+.stars--4 .stars__star:nth-child(n + 5),
+.stars--5 .stars__star:nth-child(n + 6) {
+  background-image: url(images/star.svg);
```
Comment (ivan-kadykalo) [inline src/style.css:29]: And here use active stare

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star:nth-child(n),
+.stars--1 .stars__star:nth-child(n + 2),
+.stars--2 .stars__star:nth-child(n + 3),
```
Comment (ivan-kadykalo) [inline src/style.css:25]: We can write it shorter
```suggestion
.stars--2 :nth-child(n + 3),
```

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]
Demo links are not active. Delete <> braces from the link

File: src/style.css (hunk 1 above)
Comment (Paryax) [reply src/style.css:12]: If this is done, then the asterisks will be located differently than in the task condition

**Review by ivan-kadykalo — APPROVED** [review]
🚀

**Pull request closed without merging**